pgcli/pgcompleter.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, smart_completion=True):
 
         self.special_commands = []
         self.databases = []
-        self.dbmetadata = {}
+        self.dbmetadata = {'tables': {}, 'functions': {}}
```

Comment:
I'm going to merge this in and then refactor it into a different var. Deep nested dicts make me nervous about the complexity. Even though there are comment strings, I'm afraid that we won't remember it very well when we come back to it in 6 months.